src/main/java/frc/robot/commands/BangBangRotateCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands;
+
+import frc.robot.subsystems.DriveSub;
+
+public class BangBangRotateCommand extends BangBangSubsystemCommand {
+    
+    private DriveSub subsystem;
+    private double rotations;
+
+
+
+    public BangBangRotateCommand(DriveSub subsystem, double rotations) {
+        super(subsystem, 0);
+        this.subsystem = subsystem;
+        this.rotations = rotations;
+    }
+    
+
+        public double getPoint() {
+        return rotations;
+        }
+
+    @Override
+    public void initialize() {
+      setPoint(subsystem.getPosition() + rotations * 360);
```

Comment:
Since you changed everything in DriveSub to rotations, you don't need to multiply by 360 anymore.

Reply:
fixed it

Reply:
It now uses integer division, so 1/360 = 0. To fix this, you can use 1.0/360 or Units.rotationsToDegrees().